meta-resin-common/recipes-support/resin-mounts/resin-mounts/etc-docker.mount

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Docker key.json location
```

Comment:
Maybe a little more description: Docker key.json bind mount for read-only filesystem